drivers/gnss/gnss_luatos_air530z.c

```diff
@@ -188,21 +188,21 @@ static int gnss_luatos_air530z_init(const struct device *dev)
 
 	luatos_air530z_init_dynamic_script(dev);
 
-	ret = modem_pipe_open(data->uart_pipe);
+	ret = modem_pipe_open(data->uart_pipe, K_SECONDS(10));
```

Comment:
should the number be a kconfig option? (could be on a followup)

Reply:
Agreed could be nice to have a global default defined by a Kconfig option.